walkthroughs/week-3-libraries/maps/examples/hello-world/src/main/webapp/script.js

```diff
@@ -18,3 +18,24 @@ function createMap() {
       document.getElementById('map'),
       {center: {lat: 37.422, lng: -122.084}, zoom: 16});
 }
+
+/** Adds GMaps script tag to the page. */
+function loadGoogleMapsScript(){
+  const script = document.createElement("script")
+  script.type = "text/javascript";
+
+  if (script.readyState){  //IE
+      script.onreadystatechange = function(){
```

Comment:
do you mind to explain why do we need this?

Reply:
I did it to prevent storing the API key on Github. It is stored in config.js which will be ignored by git

Reply:
I see, sounds good! 👍